[docs] [drivers] Move node.js and csharp driver pages to Drivers and ORMs. (#14689)
yugabyte/yugabyte-db: docs/content/preview/reference/drivers/ysql-client-drivers.md

For details on Npgsql, refer to the [Npgsql documentation](https://www.npgsql.org/doc/).

> To follow a tutorial on building a sample C# application with Npgsql, see [Connect an application](../../../drivers-orms/csharp/postgres-npgsql/).

Reviewer: ```suggestion
For a tutorial on building a sample C# application with Npgsql, see [Connect an application](../../../drivers-orms/csharp/postgres-npgsql/).
```
